Gems/LmbrCentral/Code/Source/Audio/EditorAudioListenerComponent.h

```diff
@@ -9,7 +9,7 @@
 #pragma once
 
 #include <AzToolsFramework/ToolsComponents/EditorComponentBase.h>
-
+#include <AzCore/Math/Vector3.h>
```

Comment:
Kinda weird that this is needed now?

Reply:
Its because I removed the hard requirement of transform.h from editorbasecomponent.h  - this file was using it from there.